packages/openapi-to-graphql/src/preprocessor.ts

```diff
@@ -155,6 +355,17 @@ export function preprocessOas(
         // Links
         const links = Oas3Tools.getEndpointLinks(path, method, oas, data)
 
+        // Callbacks containing [key: string]:PathItemObject
+        const callbacks = Oas3Tools.getEndpointCallbacks(
+          path,
+          method,
+          oas,
+          data
+        )
+
+        // should every callback items be registered as operations ?
+        processOperationCallbacks(callbacks, oas, data, options)
+
```

Comment:
Retrieve CallbacksObject from the operationObject and resolve $refs when needed.